  Running command git clone --quiet https://github.com/huggingface/peft.git /tmp/pip-install-1vkck7pb/peft_5a8baaa0c4c64d90bec02d215893f04e
  Running command git clone --quiet https://github.com/huggingface/accelerate.git /tmp/pip-install-1vkck7pb/accelerate_9dcedebfffba4ce19272cdb67d95bca7
/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/tokenization_utils_base.py:1714: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/modeling_utils.py:2193: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
Loading checkpoint shards:   0%|          | 0/15 [00:00<?, ?it/s]Loading checkpoint shards:   7%|▋         | 1/15 [00:04<01:00,  4.35s/it]Loading checkpoint shards:  13%|█▎        | 2/15 [00:07<00:45,  3.48s/it]Loading checkpoint shards:  20%|██        | 3/15 [00:09<00:37,  3.08s/it]Loading checkpoint shards:  27%|██▋       | 4/15 [00:12<00:31,  2.90s/it]Loading checkpoint shards:  33%|███▎      | 5/15 [00:15<00:27,  2.79s/it]Loading checkpoint shards:  40%|████      | 6/15 [00:17<00:24,  2.76s/it]Loading checkpoint shards:  47%|████▋     | 7/15 [00:20<00:21,  2.68s/it]Loading checkpoint shards:  53%|█████▎    | 8/15 [00:23<00:19,  2.83s/it]Loading checkpoint shards:  60%|██████    | 9/15 [00:26<00:16,  2.76s/it]Loading checkpoint shards:  67%|██████▋   | 10/15 [00:28<00:13,  2.71s/it]Loading checkpoint shards:  73%|███████▎  | 11/15 [00:31<00:10,  2.69s/it]Loading checkpoint shards:  80%|████████  | 12/15 [00:33<00:07,  2.64s/it]Loading checkpoint shards:  87%|████████▋ | 13/15 [00:36<00:05,  2.64s/it]Loading checkpoint shards:  93%|█████████▎| 14/15 [00:39<00:02,  2.71s/it]Loading checkpoint shards: 100%|██████████| 15/15 [00:39<00:00,  1.95s/it]Loading checkpoint shards: 100%|██████████| 15/15 [00:39<00:00,  2.63s/it]
WARNING:root:The `device_map` argument is not provided. We will override the device_map argument. to set the entire model on the current device. If you want to set the model on multiple devices, please provide a custom `device_map` argument.
WARNING:datasets.builder:Found cached dataset csv (/home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d)
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 409.80it/s]
WARNING:datasets.arrow_dataset:Loading cached shuffled indices for dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-789e2502999e7416.arrow
WARNING:datasets.arrow_dataset:Loading cached shuffled indices for dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-789e2502999e7416.arrow
WARNING:datasets.arrow_dataset:Loading cached shuffled indices for dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-789e2502999e7416.arrow
WARNING:datasets.arrow_dataset:Loading cached processed dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-db79731672741871.arrow
WARNING:datasets.arrow_dataset:Loading cached processed dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-4f9937f95ed3fc2e.arrow
WARNING:datasets.arrow_dataset:Loading cached processed dataset at /home/brc4cb/.cache/huggingface/datasets/csv/default-10416fda93fda02b/0.0.0/eea64c71ca8b46dd3f537ed218fc9bf495d5707789152eb2764f5c78fa66d59d/cache-499200ed12fcfae2.arrow
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
0it [00:00, ?it/s]0it [00:00, ?it/s]
Traceback (most recent call last):
  File "/gpfs/gpfs0/project/SDS/research/christ_research/Llama 2/llama2-70b/ppo.py", line 258, in <module>
    question = ppo_trainer.generate(inputs, attention_mask=attention_mask, **generation_kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/trl/trainer/ppo_trainer.py", line 454, in generate
    response = self.accelerator.unwrap_model(self.model).generate(
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/trl/models/modeling_value_head.py", line 198, in generate
    return self.pretrained_model.generate(*args, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/peft/peft_model.py", line 977, in generate
    outputs = self.base_model.generate(**kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/generation/utils.py", line 1588, in generate
    return self.sample(
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/generation/utils.py", line 2642, in sample
    outputs = self(
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/accelerate/hooks.py", line 165, in new_forward
    output = old_forward(*args, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/models/llama/modeling_llama.py", line 806, in forward
    outputs = self.model(
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/accelerate/hooks.py", line 165, in new_forward
    output = old_forward(*args, **kwargs)
  File "/home/brc4cb/.conda/envs/falcon_40B/lib/python3.9/site-packages/transformers/models/llama/modeling_llama.py", line 623, in forward
    batch_size, seq_length = input_ids.shape
ValueError: too many values to unpack (expected 2)
